fly/commands/workers.go

```diff
@@ -191,3 +194,29 @@ func (w *worker) VersionCell() ui.TableCell {
 
 	return column
 }
+
+func (w *worker) ageCell() ui.TableCell {
+	var column ui.TableCell
+
+	const minute = 60
+	const hour = minute * 60
+	const day = hour * 24
+
+	age := time.Now().Unix() - w.StartTime
+	if w.StartTime <= 0 || age < 0 {
+		column.Contents = "na"
+		column.Color = color.New(color.Faint)
+	} else {
+		if age/day > 0 {
+			column.Contents = fmt.Sprintf("%dd", age/day)
```

Comment:
can we have days and hours here? that would be consistent with what is shown in the web UI.

Reply:
I'm trying to following `kubectl`'s output format, like:

    $ bin/getpo stage
    NAME                                     READY   STATUS      RESTARTS   AGE
    atc-6d8645d695-6zmpm                     1/1     Running     1          55d
    atc-6d8645d695-bbhxs                     1/1     Running     1          55d
    atc-6d8645d695-jjtjn                     1/1     Running     1          56d
    atc-6d8645d695-x5dt2                     1/1     Running     11         55d
    health-b4dc554f-r4zcn                    1/1     Running     1          55d
    nfs-cleaner-1569232800-84xmj             0/1     Completed   0          4h56m

I think is form is short and neat. Are you ok with this? @pivotal-jamie-klassen

Reply:
ok #4485